tech.json

```diff
@@ -512,6 +512,18 @@
                 "This consists of a tight variant of mid-air Spring Ball jump, then turning off Spring Ball, then a second mid-air Spring Ball jump all while still climbing upwards.",
                 "This is typically used underwater with HiJump, but it is technically possible in lava without HiJump or to perform more than two Spring Ball jumps in acid."
               ]
+            },
+            {
+              "name": "canUnderwaterBombIntoSpringBallJump",
+              "requires": [ 
+                "canTrickySpringBallJump",
+                "h_canJumpIntoIBJ"
```

Comment:
can this be done with PBs? If so, it could just be mentioned in a fixme or if it will likely never be used, just in a devnote